index.html

```diff
@@ -12,8 +12,76 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
+  <div class="container">
 
-<body>
-  <h1>My Resume</h1>
+    <header>
+      <nav>
+        <a href="#">Home</a>
+        <a href="#">About</a>
+        <a href="#">Portfolio</a>
+        <a href="#">Contact</a>
+      </nav>
+    </header>
+
+    <section class="intro-container">
+      <div class="main-intro">
+        <img src="img/selfie.jpg" alt="Great selfie I took just for this project lol">
+        <h1>Drew Johnson - Web Dev</h1>
+        <p>Top of the line full-stack developer</p>
+      </div>
+        <p class="lorem-text">Lorem ipsum, dolor sit amet consectetur adipisicing elit. i, nisi aut!</p>
+        <div class="team-up-btn">
+          <a href="#">Let's Team Up</a>
```

Comment:
with this pattern, you have an apparent button, but the link only works if you click right on the text. An approach you might consider instead is making the a tag a block element and giving it the button styles directly